src/components/RelativeTime.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+const showMonth = (monthNum: number) => {
+  const months = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sept",
+    "Oct",
+    "Nov",
+    "Dec",
+  ];
+  return months[monthNum];
+};
+
+const getRelativeTime = (timestamp: number) => {
```

Comment:
maybe move these into a utility file

Reply:
There is styling and tests for this, plus it creates an element on the page, which is why I think that it should remain a component.  Would you want to move the whole component out into a utility file or just part of it?  Would love to pair on how to organize this.